packages/www/pages/verify.jsx

```diff
@@ -14,9 +14,28 @@ import Guides from "components/Marketing/Guides";
 const Verify = () => {
   useLoggedIn();
   const router = useRouter();
-  const { user } = useApi();
+  const { user, verify } = useApi();
   const { email, emailValidToken } = router.query;
 
+  useEffect(() => {
```

Comment:
So, in practice, we weren't validating the user's email anymore even when they did click on the link received on the email? Good thing we got this back.

Reply:
if they did click on the link received on the email, we will have the user's email validating, so emailValid will be true.